editor/js/Sidebar.Geometry.TubeGeometry.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * @author Temdog007 / http://github.com/Temdog007
+ */
+
+Sidebar.Geometry.TubeGeometry = function ( editor, object ) {
+
+	var strings = editor.strings;
+
+	var signals = editor.signals;
+
+	var container = new UI.Row();
+
+	var geometry = object.geometry;
+	var parameters = geometry.parameters;
+
+	// points
+
+	var lastPointIdx = 0;
+	var pointsUI = [];
+
+	var pointsRow = new UI.Row();
+	pointsRow.add( new UI.Text( strings.getKey( 'sidebar/geometry/tube_geometry/path' ) ).setWidth( '90px' ) );
+
+	var points = new UI.Span().setDisplay( 'inline-block' );
+	pointsRow.add( points );
+
+	var pointsList = new UI.Div();
+	points.add( pointsList );
+
+	( function () {
+
+		var points = parameters.path.points;
+		for ( var i = 0; i < points.length; i ++ ) {
+
+			var point = points[ i ];
+			pointsList.add( createPointRow( point.x, point.y, point.z ) );
+
+		}
+
+	} )();
```

Comment:
What is the purpose of placing this code section into a closure?

Reply:
Prevent changing the variable `points`. But, it isn't necessary so I'll change that.